validation/src/main/scala/hmda/validation/ValidationStats.scala

```diff
@@ -122,8 +140,8 @@ class ValidationStats extends HmdaPersistentActor {
         updateState(e)
       }
 
-    case AddSubmissionMacroStats(id, total, q070, q070Sold, q071, q071Sold, q072, q072Sold) =>
-      persist(SubmissionMacroStatsAdded(id, total, q070, q070Sold, q071, q071Sold, q072, q072Sold)) { e =>
+    case AddSubmissionMacroStats(id, total, q070, q070Sold, q071, q071Sold, q072, q072Sold, q075, q075Sold, q076, q076Sold) =>
+      persist(SubmissionMacroStatsAdded(id, total, q070, q070Sold, q071, q071Sold, q072, q072Sold, q075, q075Sold, q076, q076Sold)) { e =>
```

Comment:
That would work. It would make the parameter lists in these Stats-related objects shorter by 2. I'm happy to make that change if it seems valuable.